<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_search">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tree_list">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>71</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="ac_add"/>
    <addaction name="ac_edit"/>
    <addaction name="ac_logout"/>
    <addaction name="ac_search"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ac_add">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="delet">
   <property name="text">
    <string>delet</string>
   </property>
  </action>
  <action name="ac_edit">
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="ac_logout">
   <property name="text">
    <string>log out</string>
   </property>
  </action>
  <action name="ac_search">
   <property name="text">
    <string>search</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
